manifests/v/VisualGPS/VisualGPSView/1.1.401/VisualGPS.VisualGPSView.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: VisualGPS.VisualGPSView
+PackageVersion: 1.1.401
+PackageLocale: en-US
+Publisher: VisualGPS, LLC
+PublisherUrl: https://www.visualgps.net/
+Author: VisualGPS, LLC
+PackageName: VisualGPSView
+PackageUrl: https://www.visualgps.net/#visualgpsview-content
+License: Freeware
+ShortDescription: Copyright © 2020 VisualGPS Inc. All rights reserved - VisualGPS™ is a trademark of VisualGPS, LLC Anaheim, CA

```

Comment:
This should probably be in Copyright, not ShortDescription.

Reply:
> This should probably be in Copyright, not ShortDescription.

May you add a suggest commit?